app/controllers/assessments_controller.rb

```diff
@@ -67,11 +67,15 @@ class AssessmentsController < ApplicationController
   def index
     @is_instructor = @cud.has_auth_level? :instructor
     announcements_tmp = Announcement.where("start_date < :now AND end_date > :now",
-                                           now: Time.now)
+                                           now: Time.current)
```

Comment:
Time.current is recommended by rubocop, and 
https://stackoverflow.com/questions/12748626/when-should-datetime-now-utc-vs-time-current-utc-be-used-in-rails